content/blog/2021-08-31-out-in-the-open-august-2021.md

```diff
@@ -0,0 +1,89 @@
+   attribution: Foto von [Leyre Labarga](https://unsplash.com/@leyre_labarga) auf [Unsplash](https://unsplash.com/photos/lfyP9zTgUp0)
+
+ og_image: out-in-the-open-2021-08/header.jpg
+
+ type: blog
+---
+Stefan Kaufmann blickt in [einem neuen
+Blogpost](https://stefan.bloggt.es/2021/08/wie-apt-get-fuer-daten-knapp-12-jahre-spaeter/) auf einen
+Vortrag vom 26C3 zum Thema CKAN und offene Daten zurück. Er vergleicht die damaligen Forderungen mit
+den Entwicklungen, die seitdem gemacht (oder auch nicht gemacht) wurden. Insbesondere stellt er die
+Frage, ob es überhaupt sinnvoll ist, Lizenzen für Daten zu fordern und verlinkt [einen alten und
+wichtigen Beitrag](http://simia.net/wiki/Free_data) vom Wikidata Gründer Denny Vrandečić zum Thema
+Free Data, in dem es um die Frage geht, wie es eigentlich aus rechtlicher Sicht um Lizenzen für
+Daten steht.
+
+Je näher der September rückt, desto stärker geraten die Neuigkeiten zur Bundestagswahl in den Blick:
+Zur Wahl am 26. September 2021 steht nun eine [offene
+Linksammlung](https://hackmd.okfn.de/bundestagswahl_2021_linksammlung) bereit, die weiter
+vervollständigt werden kann und auf dem 2017 von der Open Knowledge Foundation und Code for Germany
+erstellten Wahldaten-Repository aufbaut. Weiterhin sollen auf der [offiziellen
+Seite](https://www.bundeswahlleiter.de/) des Bundeswahlleiters neben den Wahlkreisen und
+Strukturdaten bald auch Informationen über die an der Wahl teilnehmenden Parteien und Kandidat:innen
+abrufbar sein. Nur eine Schnittstelle zum automatisierten Abruf dieser Daten steht laut
+Bundeswahlleiter nicht zur Verfügung.
+
+Nach 20 Monaten Prüfung geht der Volksentscheid Berliner Transparenzgesetz endlich in die nächste
+Runde. Nachdem der Senat festgestellt hat, dass der Volksentscheid rechtlich zulässig ist, hat das
+Abgeordnetenhaus nun vier Monate Zeit, den Gesetzentwurf zu diskutieren. Mehr Informationen [gibt es
+im Blog von
+FragDenStaat](https://fragdenstaat.de/blog/2021/08/17/volksentscheid-transparenz-ist-zulassig-initiative-kritisiert-die-20-monatige-zulassigkeitsprufung-ihres-gesetzesentwurfs/).
+
+Überraschend teilte die Deutsche Bahn im Juli mit, ihre Echtzeitdaten mit Google teilen zu wollen.
+Warum ausschließlich Google Zugang zu den Daten haben solle, anstatt eine offene Schnittstelle
+anzubieten, blieb offen. Franziska Heine von Wikimedia Deutschland erklärt [in ihrem lesenswerten
+Kommentar](https://background.tagesspiegel.de/digitalisierung/mit-google-aufs-digitale-abstellgleis)
+im Tagesspiegel Background, warum die Bahn mit ihrem Plan zehn Jahre zu spät kommt und es endlich
+für alle offene Datenschnittstellen braucht.
+
+Im März berichteten wir erstmalig von der Berliner OpenStreetMap Gruppe, die sich dem Thema 
+Verkehrswende zugewandt hat. Nun ist ein weiterer
+[Artikel](https://supaplexosm.github.io/strassenraumkarte-neukoelln/posts/2021-07-18-strassenraumkarte)
+erschienen, in dem Tobias Alex zu seiner Parkraumkarte interviewt wird und die beiden in deutlich
```

Comment:
Wer sind „die beiden“?

Reply:
Hehe, Tobias interviewt hier Alex. Bin sehr dankbar für bessere Formulierungsvorschläge :)

Reply:
> Tobias interviewt hier Alex
Dann muss aber das „wird“ weg.

Reply:
Oder: in dem Alex von Tobias zu seiner Parkraumkarte interviewt wird

Reply:
```suggestion
erschienen, in dem Alex erneut von Tobias zu seiner Parkraumkarte interviewt wird und die beiden in deutlich
```

Reply:
Ich habs mal so geändert :)